projectsFunctions.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
```

Comment:
query variables can just be `$query` here like above

Reply:
fixed

Reply:
Actually not fixed as not using these functions for MVP :-)